hack/update-openapi-spec.sh

```diff
@@ -21,6 +21,11 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# STRICT_ALPHA_HANDLING ensures that the OpenAPI is updated with all APIs
+# that are intended to be removed at a particular release. This should be true by default.
+# If a new version tag was just created and you are seeing an unrelated diff when adding
+# a new API, run `STRICT_ALPHA_HANDLING=false ./hack/update-openapi-spec.sh`.
+STRICT_ALPHA_HANDLING=${STRICT_ALPHA_HANDLING:-true}
```

Comment:
I don't think we should default this to true, or master generation will break during release branch phase

Reply:
Resolved in https://github.com/kubernetes/kubernetes/pull/130320#discussion_r1980305938. Strict will only happen in alpha but not alpha.0